lightningd/json.c

```diff
@@ -347,23 +347,37 @@ void json_add_escaped_string(struct json_stream *result, const char *fieldname,
 		tal_free(esc);
 }
 
-void json_add_amount_msat(struct json_stream *result,
-			  struct amount_msat msat,
-			  const char *rawfieldname,
-			  const char *msatfieldname)
+void json_add_amount_msat_compat(struct json_stream *result,
+				 struct amount_msat msat,
+				 const char *rawfieldname,
```

Comment:
Other `json_add_*` functions have field names before the value being printed.
Mildly suggest, to follow other functions.

Reply:
I'd agree, but it's a lot of churn for a function we're going to phase out, so I think it's a bit gratuitous for now.